pallets/identity/src/types.rs

```diff
@@ -31,3 +32,15 @@ pub enum DidStatus {
     Exists,
     CddVerified,
 }
+
+/// Aggregate information about an `AccountId` in relation to an `IdentityId`.
+#[derive(Eq, PartialEq, Encode, Decode)]
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
+pub struct KeyIdData<IdentityId> {
```

Comment:
Need to update types in the `polymesh_schema.json` related to this rpc

Reply:
Done.